pkg/server/http/ipfs.go

```diff
@@ -49,13 +49,50 @@ func ipfsHandler(lassie *lassie.Lassie, cfg HttpServerConfig) func(http.Response
 		hasAccept := req.Header.Get("Accept") != ""
 		acceptTypes := strings.Split(req.Header.Get("Accept"), ",")
 		validAccept := false
+		includeDupes := true
 		for _, acceptType := range acceptTypes {
 			typeParts := strings.Split(acceptType, ";")
 			if typeParts[0] == "*/*" || typeParts[0] == "application/*" || typeParts[0] == "application/vnd.ipld.car" {
 				validAccept = true
-				break
+				if typeParts[0] == "application/vnd.ipld.car" {
```

Comment:
wew, this really does need refactoring out to a separate "accept" header decoder func

Reply:
I actually went ahead and refactored all of the ipfs handler file, but then decided this should be a seperate PR cause... lots more changes.